MIGRATION.md

```diff
@@ -0,0 +1,101 @@
+# Migration Guide
+
+This document describes the changes that need to be made to migrate from any of the 
+individual actions ([buf-setup-action][buf-setup], [buf-breaking-action][buf-breaking], [buf-lint-action][buf-lint], [buf-push-action][buf-push]).
+
+## [buf-setup-action][buf-setup]
+
+The `buf-setup-action` has been deprecated in favor of the `buf-action` action. 
+
+```yaml
+steps:
+  - uses: actions/checkout@v2
+  - uses: bufbuild/buf-setup-action@v1.45.0
+```
```

Comment:
There is no description of what this code block is (i.e. it is unclear if this is what I should be doing or shouldn't be doing now). Same comment on other sections.

It might be simpler to express these migrations as the simplest diff possible. Like this:

To migrate to `buf-action` while retaining the same behavior as the deprecated `buf-setup-action`, set the `setup_only` input to `true` and set the `version` input to the Buf CLI version you want to pin to. Here is an example:

````diff
 steps:
   - uses: actions/checkout@v4
-  - uses: bufbuild/buf-setup-action@v1.45.0
+  - uses: bufbuild/buf-action@v1
+    with:
+      version: '1.45.0'  # Set this if you want to continue pinning to a specific version. Otherwise you can leave this unset to always use the latest version of the Buf CLI.
+      setup_only: true
````

Reply:
Have moved all sections to diff style code blocks.